docs/operational_guide/multiple_m3db_clusters.md

```diff
@@ -0,0 +1,125 @@
+## Write to multiple M3DB clusters via M3Coordinator
+
+### Overview:
+
+Default M3 architecture has the M3Coordinator writing to and aggregating meterics from a single M3DB cluster. If wanting to add more than one, follow the below instructions. 
+
+Use case(s):
+- Sending metrics to different namespaces for different retention periods, etc.
+
+### Instructions: 
+
+**Note:** Adding mutliple M3DB clusters to m3coordinator using the m3aggregator requires clusterManagement
+
+**Note:** When making API requests, an environment header needs to be set. 
+
+1. Add clusterManagement to congfig file:
+
+Example of clusterManagement config:
+
+```bash
+clusterManagement:
+  etcd:
+    env: default_env
+    zone: embedded
+    service: m3db
+    cacheDir: /data/m3kv_default
+    etcdClusters:
+      - zone: embedded
+        endpoints:
+          - 10.33.131.173:2379
+          - 10.33.131.174:2379
+          - 10.33.131.175:2379
+  ``` 
+
+Example config file with clusterManagement: 
+
+```bash
+  - namespaces:
+       - namespace: 21d
+         retention: 504h
+         type: unaggregated
+     client:
+       config:
+         service:
+           env: default_env
+           zone: embedded
+           service: m3db
+           cacheDir: /data/m3kv_default
+           etcdClusters:
+             - zone: embedded
+               endpoints:
+                 - 10.33.131.173:2379
+                 - 10.33.131.174:2379
+                 - 10.33.131.175:2379
+       writeConsistencyLevel: majority
```

Comment:
prob dont need all of this other stuff? e.g. writeConsistencyLevel, readConsistencyLevel, etc.

Reply:
ok.. i just copied the one from the walmart slack channel... should i still remove these lines?

Reply:
You can remove these, they're equal to the defaults:

`writeConsistencyLevel`
`writeTimeout`
`fetchTimeout`
`connectTimeout`
`backgroundHealthCheckFailLimit`
`backgroundHealthCheckFailThrottleFactor`

This one `readConsistencyLevel` is different in this config than the default, which would be `majority`, if this isn't intentional, can also remove it